fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/expressions/GroupByExpression.java

```diff
@@ -451,7 +451,7 @@ private BooleanWithConstraint groupingSubsumedBy(@Nonnull final Quantifier candi
         //
         // Consult already bound predicates from the child match.
         //
-        final var equalityPredicates =
+        final var equalityOrIsNullPredicates =
```

Comment:
Let's revert that as all of those predicates are considered equality predicates in a sense that they only allow one value to pass the filter. Even an `IS NULL` is for these purposes a predicate that checks for the equality with `NULL`.

Reply:
Got it, reverted.